test/remote/gateways/remote_sage_pay_test.rb

```diff
@@ -159,12 +159,13 @@ def test_successful_visa_purchase
     assert !response.authorization.blank?
   end
 
-  def test_successful_maestro_purchase
-    assert response = @gateway.purchase(@amount, @maestro, @options)
-    assert_success response
-    assert response.test?
-    assert !response.authorization.blank?
-  end
+  # Maestro is not available for GBP
+  # def test_successful_maestro_purchase
+  #   assert response = @gateway.purchase(@amount, @maestro, @options)
+  #   assert_success response
+  #   assert response.test?
+  #   assert !response.authorization.blank?
+  # end
```

Comment:
@curiousepic wondering if we should uncomment these? Or if they're not needed just delete the test.